### ISSUE TITLE
"Try it now!" section breaks without trailing slash

### ISSUE THREAD

**Issue by nfiles** [phalt/swapi#77]
On the [swapi.co](https://swapi.co) site, the "Try it now!" section breaks if there is not a trailing slash on the request.

### Examples

**Works**: `people/1/`
**Doesn't Work**: `people/1`

### Investigation

Requests that do not have the trailing slash are immediately cancelled and redirected (by the browser?) to an HTTP endpoint that does have the trailing slash.

![image](https://cloud.githubusercontent.com/assets/1622383/24558671/f75152b6-1609-11e7-9c2b-c6481eb23372.png)

This HTTP request fails with the following error in the console:

```
Mixed Content: The page at 'https://swapi.co/' was loaded over HTTPS, but requested an insecure
XMLHttpRequest endpoint 'http://swapi.co/api/people/1/'. This request has been blocked; the
content must be served over HTTPS.
```

I see that you use CloudFlare, so maybe there is some configuration that controls this? I'm not sure where to start investigating an error like this.

Browser: Google Chrome Version 57.0.2987.110

**Comment by nfiles** [comment]
Nevermind, I found it.

It looks like endpoints without a trailing slash return a 301 response, but the "location" header still has the HTTP endpoint. Maybe Django needs to be configured to use the HTTPS endpoint?

**Comment by nfiles** [comment]
I believe this issue was resolved by PR #94 and Issue #90 and can be closed
